.travis.yml

```diff
@@ -1,47 +1,36 @@
 language: cpp
-sudo: false
+env:
+  global:
+    - GAPROOT=gaproot
+    - COVDIR=coverage
+    - GAP_PKGS_TO_BUILD="io" # exclude profiling, i.e., *this* package
+    - NO_COVERAGE=1  # tests are not compatible with regular coverage collection
 
-branches:
-  except:
-    - gh-pages
+addons:
+  apt_packages:
+    - libgmp-dev
+    - libreadline-dev
+    - libgmp-dev:i386
+    - libreadline-dev:i386
+    - gcc-multilib
+    - g++-multilib
 
 matrix:
   include:
-    - os: linux
-      compiler: gcc
-      env: BRANCH=master ABI=32 PKGOPTS="CFLAGS=-m32 LDFLAGS=-m32 LOPTS=-m32 CXXFLAGS=-m32"
-      addons:
-        apt_packages:
-          - libgmp-dev:i386
-          - gcc-multilib
-          - g++-multilib
-    - os: linux
-      env: BRANCH=master
-      compiler: gcc
-      addons:
-        apt_packages:
-         - libgmp-dev
-    - os: linux
-      env: BRANCH=stable-4.10
-      compiler: gcc
-      addons:
-        apt_packages:
-         - libgmp-dev
-
+    - env: GAPBRANCH=master ABI=32 PKGOPTS="CFLAGS=-m32 LDFLAGS=-m32 LOPTS=-m32 CXXFLAGS=-m32"
+    - env: GAPBRANCH=master
+    - env: GAPBRANCH=stable-4.10
 
-install:
-  - git clone --depth=2 https://github.com/gap-system/gap.git gapsrc -b ${BRANCH}
-  - pwd
-  - ( cd gapsrc && (./autogen.sh || true) && ./configure --with-gmp=system && make configure && make -j2 && make bootstrap-pkg-minimal)
```

Comment:
So one major drawback of this PR is that it switches from `bootstrap-pkg-minimal` to `bootstrap-pkg-full` which is of course slower, as more data has to be downloaded.

But I wouldn't mind adding an option (e.g. via an env var) to the [pkg-ci-scripts](https://github.com/gap-system/pkg-ci-scripts.git) to allow changing this from full to minimal for authors who want that...

Reply:
I've implemented this now, via GAP_BOOTSTRAP=minimal